charts/stable/omada-controller/values.yaml

```diff
@@ -24,7 +24,10 @@ service:
     ports:
       main:
         protocol: HTTPS
+        # Nodeport fix for Ingress users
+        nodePort: "{{ if .ingress.main.enabled }}443{{ else }}{{ .Values.service.main.ports.main.port }}{{ end }}"
```

Comment:
you probably mean `targetPort`.

`targetPort` is set to the same port as `port` 
when not specified. So it already does what you doing here